chart/Chart.yaml

```diff
@@ -3,5 +3,5 @@ name: py-kube-downscaler
 description: A Helm chart for deploying py-kube-downscaler
 
 type: application
-version: 0.2.6
-appVersion: "24.8.1"
+version: 0.2.7
+appVersion: "24.8.2"
```

Comment:
should we really make a new release for this small fix

Reply:
I think it was requested by @viktorhaid. Unfortunately I think it is necessary because it fixes regression